languages/java/exercises/concept/case-statement/.docs/hints.md

```diff
@@ -0,0 +1,13 @@
+## General
+
+[switch statement][switch-statement] documentation provides an introduction to `switch` statements.
+
+### 1. Output descriptions of the players based on their shirt number
+
+- The [`break`] statement is useful, see the doc!.
+
+### 2. Raise an alert if an unknown shirt number is encountered.
+
+- The [`default`] statement is useful, see the doc!.
```

Comment:
```suggestion
- The [`default`] statement can be used here.
```

Could you link to a resource that explains the `default` keyword?

Reply:
Done!